packages/widget-chat/src/actions/user.js

```diff
@@ -37,7 +37,7 @@ export function fetchCurrentUser(spark) {
 export function fetchUser(userEmail, spark) {
   return (dispatch) => {
     dispatch(requestUser(userEmail));
-    spark.user.getByEmail({email: userEmail})
+    spark.user.getUUID(userEmail)
       .then((user) => dispatch(receiveUser(userEmail, user)));
```

Comment:
Oh, weird... when i originally saw this, the diff looked like you removed line 41...
If we're not using the fetchUser action anywhere, then lets get rid of this.